src/utils/youtube/index.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line
+//const APIKEY = "AIzaSyB9PDUD5tOp5vzb8rPjNFD0v7XWP25hZVg";
```

Comment:
for security reasons should not leave secrets or API keys this case even commented out